testIntegration/001-user-create.test.ts

```diff
@@ -0,0 +1,24 @@
+import { UserStatus } from '../core/meta/user'
+import { UserService } from '../server/service/user'
+
+test('ExpectedUser === User', async () => {
+  const userParams = {
+    email: 'test@fra-platform.com',
+    name: 'Test User',
+  }
+
+  const user = await UserService.create({
+    user: userParams,
+  })
```

Comment:
when creating a user, we should also insert a user provider.
insert for now a provider 'local' with props {password:'test'}.
do not insert 'test', but the hashed version